cli/integrationtest/controlplane_test.go

```diff
@@ -2404,6 +2405,86 @@ func TestServerLogs(t *testing.T) {
 	require.Contains(t, lines[0], `"timestamp"`)
 }
 
+func TestServerApiVersioningErrors(t *testing.T) {
+	t.Parallel()
+
+	bufferId := runTygerSucceeds(t, "buffer", "create")
+
+	resp, err := controlplane.InvokeRequest(context.Background(), http.MethodGet, fmt.Sprintf("/buffers/%s?api-version=0.1", bufferId), nil, nil, nil)
+	require.Error(t, err)
+	require.Equal(t, http.StatusBadRequest, resp.StatusCode)
+	require.Contains(t, err.Error(), "Unsupported API version")
```

Comment:
Perhaps a nice thing to do would be to include the supported API versions in the reponse. This way, if we ever wanted to do fallback behavior in the client, we could save a call to the metadata endpoint.